jetstream/analysis.py

```diff
@@ -629,3 +633,47 @@ def ensure_enrollments(self, current_date: datetime) -> None:
             )
         except Conflict:
             pass
+
+    def export_errors(self, bucket_name: str):
```

Comment:
Maybe we should move this into the `export_json` module?

Reply:
Sounds like a reasonable spot for this to be. I was mirroring the approach used with `export_metadata` living in `metadata` module, but this isn't quite the same (and maybe it'd be appropriate to move that function as well).